scripts/setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Use ./script/setup.sh to install dependencies. Eventually we should also include setting up a virtual env.